python/ray/autoscaler/_private/util.py

```diff
@@ -384,13 +384,35 @@ def fill_node_type_min_max_workers(config):
                 node_type_data.setdefault("max_workers", global_max_workers)
 
 
+def with_envs(cmds, kv=Dict[str, str]) -> str:
```

Comment:
```suggestion
def with_envs(cmds: List[str], kv:Dict[str, str] = None) -> str:
```

We cannot use the mutable value as the default. https://www.codecademy.com/learn/learn-intermediate-python-3/modules/int-python-function-arguments/cheatsheet